sdk/core/System.ClientModel/src/Message/ArrayBackedRequestHeaders.cs

```diff
@@ -19,6 +19,9 @@ public override void Set(string name, string value)
 
     public override void Add(string name, string value)
     {
+        Argument.AssertNotNull(name, nameof(name));
```

Comment:
Do we want to allow `string.Empty` for these?   I don't believe that's a valid for an HTTP header, is it?

Reply:
Addressed.